angular/projects/spark-angular/src/lib/directives/sprk-link/sprk-link.stories.ts

```diff
@@ -81,37 +87,77 @@ export const simple = () => ({
 
 simple.story = {
   parameters: {
-    jest: [
-      'sprk-link.directive',
-    ],
+    jest: ['sprk-link.directive'],
+  },
+};
+
+export const light = () => ({
+  moduleMetadata: modules,
+  template: `
+    <div sprkBox [ngStyle]="{'background-color':'#1c1b1a'}">
+      <a
+        sprkLink
+        href="#"
+        variant="light"
+        idString="light-link"
+        analyticsString="light-link"
+      >
+        Light Link
+      </a>
+    </div>
+  `,
+});
+
+light.story = {
+  parameters: {
+    jest: ['sprk-link.directive'],
   },
 };
 
 export const iconWithTextLink = () => ({
   moduleMetadata: modules,
   template: `
-    <a
-      href="#"
-      sprkLink
-      variant="icon"
-      idString="icon-link"
-      analyticsString="icon-link"
-    >
-      <sprk-icon
-        iconType="communication"
-        additionalClasses="sprk-c-Icon--l sprk-c-Icon--stroke-current-color sprk-u-mrs"
+    <div>
```

Comment:
Instead of laying these out with an empty `div` and `!important` utility classes we could use Stack/Split with small spacing to achieve the same.